src/components/Switch/Switch.jsx

```diff
@@ -0,0 +1,121 @@
+import _ from 'lodash';
+import classNames from 'classnames';
+import React from 'react';
+
+import { bindClassNames } from '../../util/style-helpers';
+
+const generateBoundClassNames = bindClassNames('Switch');
+const {
+	bool,
```

Comment:
Those are tabs in my code so it might be an issue with whatever is rendering the code on the GitHub page.